foundations/Cargo.toml

```diff
@@ -77,6 +77,12 @@ telemetry = [
     "telemetry-server",
 ]
 
+tokio-metrics = [
```

Comment:
Let's rename it to `tokio-runtime-metrics`

Reply:
and the module as well

Reply:
No objections here, I don't really care all that much what color we paint the bikeshed in this case.